incubator/kafka/README.md

```diff
@@ -175,3 +187,32 @@ such port at a time, setting the range at every Kafka pod is a reasonably safe c
 * There must not exist a service called `kafka` in the same namespace
 
 [brokerconfigs]: https://kafka.apache.org/documentation/#brokerconfigs
+
+## Prometheus Stats
+
+### Prometheus vs Prometheus Operator
+
+Prometheus is default option of this chart b/c the Prometheus Operator is part of another repo: 
+
+https://github.com/coreos/prometheus-operator/tree/master/helm 
+
+The Prometheus Operator gives additional functionality like automagically updating Prometheus and Alert Manager configuration.  These are things you will
+need to manually do if you are running Prometheus without the Operator.  For more details about the Prometheus Operator:
+
+https://coreos.com/blog/the-prometheus-operator.html
+
+### JMX Exporter
+
+https://github.com/prometheus/jmx_exporter Almost all the stats on Kafka come from JMX.  There are some limited complimentary stats that come from
+Kafka Exporter.  JMX Exporter is a generic exporter not only intended for Kafka.  When using with Kafka you get way more stats than one could want,
+white list them all or connect to JMX port (not Prometheus JMX Port used for scraping), also can see raw stats that way.
+
+JMX performs some transforms but without configuration metrics are not in a Prometheus format, things like broker names and topic names become part of
+the metric name instead of being a label.  Further things that should be part of the metric name show as labels.  Metrics have been cleaned up in default
+JMX ConfigMap to be in a proper Prometheus format.  As more metrics are cleaned up, they should be exposed as well.  Then users can control which metrics
+show via `whitelistObjectNames`.  PR's are gladly accepted.
+
+### Kafka Exporter
+
+https://github.com/danielqsj/kafka_exporter is complimentary to JMX Exporter.  It gives additional stats on things like Consumer Groups, see
```

Comment:
The [Kafka Exporter](https://github.com/danielqsj/kafka_exporter) is a complimentary metrics exporter to the JMX Exporter. The Kafka Exporter provides additional statistics on Kafka Consumer Groups.